docs/prepare-your-dev-environment.md

```diff
@@ -10,10 +10,21 @@ This document goes through the development dependencies one requires in order to
 
 1. Configure `GOPATH` as an OS environment variable in your shell (a requirement of some dependencies for `make generate`). If you want to keep the default path, you can add something like `GOPATH=$(go env GOPATH)` to your shell's profile/RC file.
 
+<<<<<<< HEAD
```

Comment:
looks like this bit snuck in with a merge conflict

Reply:
argh. fixed.